oi/type_graph/Flattener.cpp

```diff
@@ -152,6 +191,9 @@ void Flattener::visit(Class& c) {
     flattenParent(parent, flattenedMembers);
   }
 
+  // Perform fixups for bad DWARF
+  fixAllocatorParams(c);
```

Comment:
Would it be nicer to check it's an allocator here? It doesn't feel like very natural to call `fixAllocatorParams` on things which aren't allocators, unless it's a pass of its own.

Reply:
I see what you mean, but I personally have a preference for minimising indentation levels and hiding away complexity.

I was envisioning that in the future there might be various fix-ups we need to do for various kinds of bad DWARF, so we could end up with some code like:
```
fixAllocatorParams(c);
fixSomethingElse(c);
fixAnotherThing(c);
```

Which I think would be cleaner and easier to read at a glance than the alternative:
```
if (...) {
  fixAllocatorParams(c);
}
if (...) {
  fixSomethingElse(c);
}
if (...) {
  fixAnotherThing(c);
}
```